pages/linux/raw.md

```diff
@@ -0,0 +1,8 @@
+# raw
+
+> Bind a raw character device.
```

Comment:
What exactly is a raw character device?

Reply:
https://tldp.org/HOWTO/SCSI-2.4-HOWTO/rawdev.html

RTFM.

Reply:
No need for that.

Reply:
I've now read that @pixelcmtd and I still don't understand.

Reply:
$ tldr raw device

**raw device**

A way of performing raw (i.e. unbuffered) I/O on a block device.

- I don't need no examples.

Reply:
What about being a little more specific? 
```suggestion
> Bind a Unix raw character device.
```

Reply:
I see no harm in doing so.